community-new.html

```diff
@@ -871,13 +871,13 @@ <h4 id="host-a-ballerina-event-title" class="gTitle1">Host a Ballerina event</h4
        <p>Want to talk about Ballerina at your local tech meetup? Reach us at <a class="getStartLinks" href="mailto:contact@ballerina.io">contact@ballerina.io</a>, and we will help you with any presentation content.</p>
            
         </div>
-        <div class="col-lg-4 col-md-6 col-sm-12 card" style="margin-right:0px !important;">
+        <!-- <div class="col-lg-4 col-md-6 col-sm-12 card" style="margin-right:0px !important;">
```

Comment:
can we just remove this instead of commenting?

Reply:
Done.